compiler/src/iree/compiler/Codegen/LLVMGPU/test/create_tile_sizes.mlir

```diff
@@ -0,0 +1,24 @@
+// RUN: iree-opt -iree-transform-dialect-interpreter %s | FileCheck %s
+
+func.func @matmul(%lhs: tensor<128x128xf16>, %rhs: tensor<128x128xf16>) -> tensor<128x128xf32> {
```

Comment:
We usually add `CHECK` or `CHECK-LABEL` about `func.func @matmul`. Because it moves the curser to the start point. It is probably fine if we only have a single test in this file, but I think it is still better to add such check.

```suggestion
// CHECK-LABEL: func.func @matmul
func.func @matmul(%lhs: tensor<128x128xf16>, %rhs: tensor<128x128xf16>) -> tensor<128x128xf32> {
```